### PR TITLE
Install and Usage example in TS

### PR DESCRIPTION
**Pull request by gfviegas**
Simple install and usage in Typescript.

### CODE REVIEW

**Comment by maggiepint** [comment]
THANK YOU! I was going to have to learn TypeScript for this. You just saved me a lot of time.

File: docs/moment/00-use-it/10-typescript.md [hunk 1]
```diff
@@ -5,4 +5,33 @@ version: 2.13.0
...
+
+
+let now = moment().format('LLLL');
+```
+
+**Note:** If you have trouble importing moment, try add ```"allowSyntheticDefaultImports": true``` in ```compilerOptions``` in your ```tsconfig.json``` file.
+
+**Locale Import**
+
+To use `moment.locale` you need first to import the language you are aiming for otherwise it'll keep in the current language.
```
Comment (maggiepint) [inline docs/moment/00-use-it/10-typescript.md]: Minor grammatical error. Can we say:

To use `moment.locale` you first need to import the language you are targeting.

Comment (gfviegas) [inline docs/moment/00-use-it/10-typescript.md]: Fixed.

**Comment by ichernev** [comment]
Merged in 259c0bf

**Pull request closed without merging**